instrumentation/opentelemetry-instrumentation-falcon/src/opentelemetry/instrumentation/falcon/__init__.py

```diff
@@ -297,9 +323,17 @@ def process_response(
                     description=reason,
                 )
             )
+
+            # Falcon 1 does not support response headers. So
+            # send an empty dict.
+            if _falcon_version == 1:
+                _response_headers_dict = {}.items()
+            else:
+                _response_headers_dict = resp.headers.items()
+
             if span.is_recording() and span.kind == trace.SpanKind.SERVER:
                 otel_wsgi.add_custom_response_headers(
-                    span, resp.headers.items()
+                    span, _response_headers_dict
```

Comment:
nit: this is list
```suggestion
                    span, _response_headers_list
```

Reply:
Refactored here. Its technically not a dict or list. Its a dict_list. Moved the .items() back to where it was to make it more clear. (hopefully)